flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveDynamicTableFactory.java

```diff
@@ -114,17 +117,19 @@ public DynamicTableSource createDynamicTableSource(Context context) {
                                                     STREAMING_SOURCE_PARTITION_INCLUDE.key(),
                                                     STREAMING_SOURCE_PARTITION_INCLUDE
                                                             .defaultValue()));
+            JobConf jobConf = new JobConf(hiveConf);
+            addCredentialsIntoJobConf(jobConf);
```

Comment:
Can we merge these into a single util method? Something like `JobConfUtils::createJobConfWithCredentials`

Reply:
Got it.